apps/tup-cms/src/taccsite_cms/secrets.sample.py

```diff
@@ -1 +1,4 @@
 TUP_SERVICES_ADMIN_JWT = "CHANGEME"
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = ""
```

Comment:
**For Future Readers.** Don't do this. [It will get you in trouble.](https://tacc-main.atlassian.net/wiki/spaces/~wbomar/pages/7279974/Django+CMS+-+Developer+Guide+-+Send+E-Mail)

Reply:
- https://github.com/TACC/tup-ui/pull/441